libs/utils/src/scripts/generate_env_file.ts

```diff
@@ -1,4 +1,3 @@
-/* c8 ignore file */
```

Comment:
An unneeded ignore, given this line in the `libs/utils` `jest.config.js`: https://github.com/votingworks/vxsuite/blob/3ba81d4d418df406aa5304fac2dbbc6e9cf159ae/libs/utils/jest.config.js#L24